tests/components/sensor/test_pilight.py

```diff
@@ -0,0 +1,119 @@
+"""The tests for the Pilight sensor platform."""
+import unittest
+from unittest.mock import patch
+
+from homeassistant.bootstrap import _setup_component
+import homeassistant.components.sensor as sensor
+from homeassistant.components import pilight
+
+from tests.common import get_test_home_assistant
+
+
+def fire_pilight_message(hass, protocol, data):
+    """Fire the fake pilight message."""
+    hass.bus.fire(pilight.EVENT, {
+        pilight.ATTR_PROTOCOL: protocol,
+        **data
```

Comment:
[PEP448]( https://www.python.org/dev/peps/pep-0448/) and python 3.4... Have to do that differently.